regression-tests/mixed-fixed-type-aliases.cpp2

```diff
@@ -20,5 +21,5 @@ main: (args) -> int = {
     test(z);
 
     for args do (arg)
-        std::cout << arg << "\n";
+        std::cout << std::filesystem::path(arg).filename() << "\n";
```

Comment:
This is a bit of a hack, but avoids issues like different relative paths or absolute path used for execution.